src/FHIR/SMART/ClientAdminController.php

```diff
@@ -130,12 +147,68 @@ public function editAction($clientId, $request)
             $this->notFoundAction($request);
             return;
         }
+        $trustedUserService = new TrustedUserService();
+        $trustedUsers = $trustedUserService->getTrustedUsersForClient($clientId);
+
+        $usersWithAccessTokens = [];
+        $userService = new UserService();
+        $patientService = new PatientService();
+        foreach ($trustedUsers as $user) {
+            // TODO: do we need to optimize this query?
+            if (UuidRegistry::isValidStringUUID($user['user_id'])) {
+                $registryRecord = UuidRegistry::getRegistryRecordForUuid($user['user_id']);
```

Comment:
![2DV](https://user-images.githubusercontent.com/278968/128997318-9a760205-8d47-49d6-8453-7442da7848f9.gif)

Reply:
This is a neat use of the uuid registry table. But couldn't you just search directly in the patient_data table uuid column?

Reply:
@bradymiller Problem is that the user_id can  beither a patient_data.uuid or a users.uuid so I would have to search both tables first.  That's why I hit the registry to figure out what kind of uuid it is.